colabs/prompts/W&B_LLM_QA_Bot.ipynb

```diff
@@ -0,0 +1,681 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.        project_name: str = "gradient_dissent_qabot"
</pre><p>This hard coding is not the same as the above dynamic project name</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/461/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>